packages/patternfly-4/react-table/src/components/Table/utils/decorators/cellActions.tsx

```diff
@@ -17,24 +18,24 @@ export default (actions, actionResolver, areActionsDisabled) => (
       extraParams: { dropdownPosition, dropdownDirection }
     },
     property
-  }
+  } : IExtra
 ) => {
   const extraData = {
     rowIndex,
     columnIndex,
     column,
     property
   };
-  const resolvedActions = resolveOrDefault(actionResolver, actions, rowData, extraData);
+  const resolvedActions = resolveOrDefault(actionResolver, actions, rowData, extraData) as IAction[];
   const resolvedIsDisabled = resolveOrDefault(
     areActionsDisabled,
     rowData && rowData.disableActions,
     rowData,
     extraData
-  );
+  ) as boolean;
```

Comment:
It seems like it's possible for this to be falsy but not `false`, which could throw an unnecessary TypeError, right? Or does `as boolean` actually cast to a boolean? (I need to read up on TS a bit more)

Reply:
again, believe we are just giving static type return types - this won't change the existing logic, only attempt to give static analysis hints.